internal/app/options/flags.go

```diff
@@ -130,6 +130,72 @@ func GetFlagsAndBeforeFunc() ([]cli.Flag, cli.BeforeFunc) {
 			Usage:   "specify the path of the config file",
 		},
 		cli.VersionFlag,
+		altsrc.NewIntFlag(&cli.IntFlag{
+			Name:  "num-namespaces",
+			Usage: "maximum number of namespaces that can be copied. Recommended to keep this number under 15 to avoid performance issues. Defaults to 8.",
+			Required: false,
+			Hidden: true,
+		}),
+		altsrc.NewIntFlag(&cli.IntFlag{
+			Name:  "server-timeout",
```

Comment:
If you are only using the setting for Cosmos DB, then let's preface the name with "cosmos", otherwise it's unclear